src/box/lua/upgrade.lua

```diff
@@ -1238,6 +1260,8 @@ end
 
 local function upgrade_to_2_11_0()
     revoke_write_access_on__collation_from_role_public()
+    -- Revoke again if upgrade to 2.10.4 was skipped.
+    revoke_execute_access_to_lua_function_from_role_public()
```

Comment:
I don't think we need to call this function here - it should be enough to call it only in `upgrade_to_2_10_4()`, because 2.11 hasn't been released yet so upgrade to 2.10.4 can't possibly be skipped.

Reply:
Oh, removed.